gui/widgets/qopenglmousetracker.hpp

```diff
@@ -28,7 +28,16 @@ class QOpenGLMouseTracker : public QOpenGLWidget {
   QVector3D GetCameraPosition() const;
   QVector3D GetCameraOrientation() const;
 
-  void addMesh(const rendering::SMesh &mesh, QColor color);
+  /**
+   * @brief Number of visible submeshes is influenced by the number of colors.
+   *
+   * The colors are distributed to submeshes in the same order they are
+   * provided. In case of a smaller number of colors, submeshes with higher
+   * index will not be visible.
+   */
+
+  void SetSubMeshes(const sme::mesh::Mesh3d &mesh,
+                    const std::vector<QColor> &colors = std::vector<QColor>(0));
```

Comment:
Here it would be nice to by default use the colours from the mesh unless otherwise specified